.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -146,6 +146,12 @@ jobs:
                 popd
               fi
             fi
+            if [ $(GROUP_NAME) == mellonax ]; then
+              make $BUILD_OPTIONS SAITHRIFT_V2=y ENABLE_SYNCD_RPC=y target/docker-saiserverv2-mlnx.gz
+              pushd ./src/sonic-sairedis/SAI
+              git stash
+              popd
```

Comment:
Why we need to do it? Could you please make it clean for the target target/docker-saiserverv2-mlnx.gz?

Reply:
resolved in https://github.com/opencomputeproject/SAI/pull/1740